package-lock.json

```diff
@@ -2753,11 +2720,6 @@
         "inherits": "2.0.3"
       }
     },
-    "bluebird": {
```

Comment:
Interesting that bluebird is removed when we remove auth.net

Reply:
Only this older version (2.11) is removed. There's still a bunch of references to a more recent version (3.5.1)